### PR TITLE
MRG: add wheel builds to release procedure

### PR DESCRIPTION
**Pull request by matthew-brett**
Document procedure for building and uploading wheels with release.

### CODE REVIEW

File: docs/development/releasing.rst [hunk 1]
```diff
@@ -155,16 +155,39 @@ packages that use the full bugfix/maintenance branch approach.)
...
-   made for release and removing the tag you created::
+#. Build and test the Astropy wheels.  See the `wheel builder README
+   <https://github.com/MacPython/astropy-wheels>`_ for instructions.  In
+   summary, clone the wheel-building repo, edit the ``.travis.yml`` and
+   ``appveyor.yml`` text files with the branch or commit for the release,
+   commit and then push back up to github.  This will trigger a wheel build
+   and test on OSX, Linux and Windows. Check the build has passed on on the
+   Travis-CI interface at https://travis-ci.org/MacPython/astropy-wheels.
+   You'll need commit privileges to the ``astropy-wheels`` repo; ask Tom Kooij
+   or on the mailing list of you do not have them.
```
Comment (bsipocz) [inline docs/development/releasing.rst]: typo, "of"-->"if"

File: docs/development/releasing.rst [hunk 2]
```diff
@@ -155,16 +155,39 @@ packages that use the full bugfix/maintenance branch approach.)
...
+
+#. If all the tests do *not* pass, you'll have to fix whatever the problem is.
```
Comment (bsipocz) [inline docs/development/releasing.rst]: Remove "all" to keep the original version

**Comment by bsipocz** [comment]
Thanks @matthew-brett. If the produce of wheels will become part of the release procedure, could you please add write access to the release coordinators? (@eteq, @astrofrog and myself, @bsipocz)?

**Comment by matthew-brett** [comment]
I've added you three to the repo.

**Comment by bsipocz** [comment]
@matthew-brett - Thanks!

**Comment by bsipocz** [comment]
This looks good, and cannot spot anything that could be an issue for sphinx, thus merging without waiting for travis. Thanks @matthew-brett!

**Pull request merged**